test/multiverse/suites/padrino/Envfile

```diff
@@ -5,8 +5,10 @@
 
 instrumentation_methods :chain, :prepend
 
+# TODO: sinatra 3's CALLERS_TO_IGNORE array is frozen, causes issues
 gemfile <<-RB
   gem 'activesupport', '~> 3'
+  gem 'sinatra', '< 3'
```

Comment:
I'm surprised this diff still shows this as new even though you merged the padrino PR already, any ideas why?

Reply:
Yeah, this was the very first attempt at fix and it was slightly different. The final version that is already in dev has "sinatra" correctly listed in alphabetical order whereas it is not in order here. I'll make sure dev keeps the right version after everything is merged.